packages/amplify-graphql-transformer-interfaces/API.md

```diff
@@ -760,6 +760,7 @@ export type TransformParameters = {
     shouldDeepMergeDirectiveConfigDefaults: boolean;
     disableResolverDeduping: boolean;
     sandboxModeEnabled: boolean;
+    allowDestructiveGraphQLSchemaUpdates: boolean;
```

Comment:
nit on naming. Other fields do not capitalize "QL" in "GraphQL". `allowDestructiveGraphqlSchemaUpdates`

Reply:
Mark this as TODO and will address in follow up PR